es-mode.el

```diff
@@ -438,10 +438,12 @@ in which case it prompts the user."
  completion, if available."
   (interactive (list 'interactive))
   (cl-case command
-    (prefix (let ((sym (company-grab-symbol)))
-              (if (string-match "\"\\(.*\\)\"?" sym)
-                  (match-string 1 sym)
-                sym)))
+    (interactive (company-begin-backend 'company-css))
```

Comment:
That’s totally a copy-paste error, so sorry about that. I updated the pull request.

I am not totally sure, but I believe the ‘interactive’ command is used when the user invoke the backend manually, for example when he only want completions from a single backend.

> On May 5, 2015, at 23:52, Lee Hinman <notifications@github.com> wrote:
> 
> In es-mode.el <https://github.com/dakrone/es-mode/pull/18#discussion_r29688497>:
> 
> > @@ -438,10 +438,12 @@ in which case it prompts the user."
> >   completion, if available."
> >    (interactive (list 'interactive))
> >    (cl-case command
> > -    (prefix (let ((sym (company-grab-symbol)))
> > -              (if (string-match "\"\\(.*\\)\"?" sym)
> > -                  (match-string 1 sym)
> > -                sym)))
> > +    (interactive (company-begin-backend 'company-css))
> Out of curiosity, why does this reference company-css which looks like the completion for css-mode?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/dakrone/es-mode/pull/18/files#r29688497>.
>